tests/integration/targets/elasticache/tasks/main.yml

```diff
@@ -0,0 +1,121 @@
+---
+
+- name: Integration testing for the elasticache module
+  module_defaults:
+    group/aws:
+      aws_access_key: '{{ aws_access_key }}'
+      aws_secret_key: '{{ aws_secret_key }}'
+      security_token: '{{ security_token | default(omit) }}'
+      region: '{{ aws_region }}'
+  collections:
+    - amazon.aws
+  block:
+    # == Dependency setup ==
+
+    - name: Create VPC to launch Elasticache instances into
+      amazon.aws.ec2_vpc_net:
+        name: elasticache-test-vpc
```

Comment:
Not sure if I need prefixing in my test resource names so they don't conflict with other tests. Also if there is s.th. like a global VPC that can be used by all tests, I am happy to use that. Just wasn't sure were to look these things up.

Reply:
The worst thing that can happen is, that your integration test will run twice at the same time, that is triggered by another PR later. Think of the other test will delete the vpc resource while the other is still using it.    
Take a look at the integration test of `ec2_instance` for example. It is using: `vpc_name: '{{ resource_prefix }}-vpc'`  
I guess it comes from `meta/` that includes `prepare_tests (_but I'm not 100% sure about it_). Maybe @tremble an clearify it.

Reply:
Ok, will just use that, seems like a common pattern at least ;)

Reply:
Yeah, you'll probably hit weird race conditions with the way this is currently laid out.

The way I tend to do it
https://github.com/ansible-collections/community.aws/blob/main/tests/integration/targets/ec2_vpc_endpoint/tasks/main.yml
https://github.com/ansible-collections/community.aws/blob/main/tests/integration/targets/ec2_vpc_endpoint/defaults/main.yml
Tries to ensure that neither the name nor the CIDR should be the same between any 2 test runs that might be happening in parallel